### PR TITLE
Upgrade rancher to version 0.5.0

### PR DESCRIPTION
**Pull request by neilbmclaughlin**
Happy with just two reviews. 2nd reviewer to merge.

### CODE REVIEW

**Review by beseven — APPROVED** [review]
I'll leave this one to @adamgraves-choices as not familiar with the rancher CLI commands/versions. Looks ok from a layman POV.

File: install_rancher.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+declare -r RANCHER_CLI_VERSION='v0.5.0'
+declare OS; OS=$(uname | tr '[:upper:]' '[:lower:]')
+declare -r URL="https://github.com/rancher/cli/releases/download/${RANCHER_CLI_VERSION}/rancher-${OS}-amd64-${RANCHER_CLI_VERSION}.tar.gz"
+declare BIN="${1:-./bin}"
```
Comment (st3v3nhunt) [inline install_rancher.sh:6]: Is it worth documenting the ability to install to another dir anywhere?

**Review by st3v3nhunt — CHANGES_REQUESTED** [review]
Should the `bin/rancher` directory be ignored? i.e. does `.gitignore` need `bin/` adding to it?

**Comment by neilbmclaughlin** [comment]
Deferring upgrade of rancher for now.

**Pull request closed without merging**